blocks/cards/cards.css

```diff
@@ -73,6 +73,10 @@ main .block.cards {
   padding: 20px;
 }
 
+.cards.hide-body .cards-card-body {
+  display: none;
```

Comment:
Does it work if we do not give any body in the word doc itself? Basically, a cards block with a single column with images. If yes, we may not need this variant.

Reply:
In our case, we need this to specify the Hyperlink but do not want this to be visible.

https://issue-280-v6--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=5